concordium-node/src/bin/cli.rs

```diff
@@ -398,12 +398,14 @@ fn start_consensus_message_threads(
         'outer_loop: loop {
             exhausted = false;
             // Update size of queues
-            node_ref.stats.set_inbound_low_priority_consensus_size(
-                consensus_receiver_low_priority.len() as i64,
-            );
-            node_ref.stats.set_inbound_high_priority_consensus_size(
-                consensus_receiver_high_priority.len() as i64,
-            );
+            node_ref
+                .stats
+                .inbound_low_priority_message_queue_size
+                .set(consensus_receiver_low_priority.len() as i64);
```

Comment:
What will happen if a potential overflow happens? Shouldn't these type conversions be saturated?

Reply:
And even if it does happen the metric would just become negative for a bit. Nothing catastrophic.

Again, in 106 thousand years :P